backend/battling/api/serializers.py

```diff
@@ -66,3 +79,80 @@ def create(self, validated_data):
 
         set_up_battle_teams_and_send_invite_email(instance)
         return instance
+
+
+class BattleTeamSerializer(serializers.ModelSerializer):
```

Comment:
I don't see a problem with having separated `read` and `write` serializers. But they need to be named accordingly and have the logic splitted. What was the reason for this change?